feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -245,19 +245,22 @@ export class AddToCartComponent implements OnInit, OnDestroy {
 
   onPickupOptionsCompLoaded() {
     if (this.featureConfigService.isEnabled('a11yPickupOptionsTabs')) {
-      this.subscription.add(
-        this.pickupOptionCompRef.instance.intendedPickupChange.subscribe(
-          (
-            intendedPickupLocation:
-              | { pickupOption?: 'pickup' | 'delivery'; displayName?: string }
-              | undefined
-          ) => {
-            this.disabled =
-              intendedPickupLocation?.pickupOption === 'pickup' &&
-              !intendedPickupLocation.displayName;
-          }
-        )
-      );
+      // Nesting this `if` statement for keeping feature flag check separately for easier removing
+      if (this.pickupOptionCompRef instanceof ComponentRef) {
```

Comment:
This change is not directly related to the issue, but still its better to have this fix as soon as possible, since there is an error in console for b2b